src/Data/List/Base.agda

```diff
@@ -196,6 +196,14 @@ tails : List A → List (List A)
 tails []       = [] ∷ []
 tails (x ∷ xs) = (x ∷ xs) ∷ tails xs
 
+insertAt : (xs : List A) → Fin (suc (length xs)) → A → List A
```

Comment:
`insertAt` should come after `updateAt` and be implemented in terms of it as is done in `Vec.Base`

Reply:
And such appeal to DRY would have avoided the detour via the previous troublesome 3-clause definition of `insertAt`.

Reply:
Wait: `updateAt` doesn't disturb the underlying *structure* of the `Vec`, it simply modifies elements. `insertAt` *does* change the structure!

Reply:
Yes, good spot. This was a nonsense comment. I was getting confused between `setAt` and `insertAt`